app/src/main/java/com/example/imageEditor/MainActivity.kt

```diff
@@ -1,11 +1,31 @@
 package com.example.imageEditor
 
-import android.os.Bundle
-import androidx.appcompat.app.AppCompatActivity
+import android.util.Log
+import com.example.imageEditor.base.BaseActivity
+import com.example.imageEditor.databinding.ActivityMainBinding
+import com.example.imageEditor.model.CollectionModel
+import com.example.imageEditor.repository.HomeRepository
+import com.example.imageEditor.ui.MainContract
+import com.example.imageEditor.ui.MainPresenter
 
-class MainActivity : AppCompatActivity() {
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+class MainActivity : BaseActivity<ActivityMainBinding>(), MainContract.View {
+    private val mainPresenter by lazy {
+        MainPresenter(HomeRepository.getInstance(this))
+    }
+
+    override fun getViewBinding(): ActivityMainBinding {
+        return ActivityMainBinding.inflate(layoutInflater)
+    }
+
+    override fun initView() {
+        mainPresenter.setView(this)
+        mainPresenter.getCollections(0)
+    }
+
+    override fun initListener() {
+    }
+
+    override fun setupCollections(list: List<CollectionModel>) {
+        Log.e(">>>>>>>", list.toString())
```

Comment:
```suggestion
        Log.d(">>>>>>>", list.toString())
```
Debug nên là Log.d e